fdbclient/include/fdbclient/MultiVersionTransaction.h

```diff
@@ -684,9 +707,15 @@ class MultiVersionTransaction : public ITransaction, ThreadSafeReferenceCounted<
 	                                               const KeySelectorRef& end,
 	                                               const StringRef& mapper,
 	                                               GetRangeLimits limits,
-	                                               int matchIndex,
 	                                               bool snapshot,
 	                                               bool reverse) override;
+	ThreadFuture<MappedRangeResult> getMappedRangeOptionalIndex(const KeySelectorRef& begin,
```

Comment:
IIRC, there was a concern that `MappedRangeResult` would also differ between the 7.1 version and the 7.2 version as the 7.2 version would have a new field. Is this not true any more, so now both the 7.1 and 7.2 versions can return the same values?

Reply:
yes, that is no longer needed so the same type of value is returned.